mmv1/templates/terraform/examples/vertex_ai_model_upload_predict_schemata.tf.erb

```diff
@@ -0,0 +1,34 @@
+resource "google_vertex_ai_models" "<%= ctx[:primary_resource_id] %>" {
+  display_name = "<%= ctx[:vars]['display_name'] %>"
+  description  = "basic upload model"
+
+  predict_schemata {
+    instance_schema_uri   = "gs://cloud-samples-data/vertex-ai/google-cloud-aiplatform-ci-artifacts/models/iris_xgboost/" # value needed
+    parameters_schema_uri = "gs://cloud-samples-data/vertex-ai/google-cloud-aiplatform-ci-artifacts/models/iris_xgboost/" # value needed
+    prediction_schema_uri = "gs://cloud-samples-data/vertex-ai/google-cloud-aiplatform-ci-artifacts/models/iris_xgboost/" # value needed
+  }
```

Comment:
Is this definitely the right type of value to pass in here?

This directoory contains only a .bst file : https://console.cloud.google.com/storage/browser/cloud-samples-data/vertex-ai/google-cloud-aiplatform-ci-artifacts/models/iris_xgboost;tab=objects?prefix=&forceOnObjectsSortingFiltering=false

Given the error:

```
  "error": {
    "code": 400,
    "message": "List of found errors:\t1.Field: model.predict_schemata.prediction_schema_uri; Message: Invalid YAML file for prediction schema.\t2.Field: model.predict_schemata.instance_schema_uri; Message: Invalid YAML file for instance schema.\t3.Field: model.predict_schemata.parameters_schema_uri; Message: Invalid YAML file for parameters schema.\t",
```

it seems that the API is expecting a YAML file. The [documentation here](https://cloud.google.com/vertex-ai/docs/reference/rest/v1/PredictSchemata) says that the value should be a URI to a YAML file

Reply:
In this case it isn't. I've searched for an example yaml file that google might provide but didn't come across any. I can make this test depend on a yaml file being uploaded before the model itself is uploaded.